bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/americanexpress/model/AccountActivity.java

```diff
@@ -2,13 +2,17 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class AccountActivity {
     List<Transaction> transactionList;
 
     public List<Transaction> getTransactionList() {
+        if(transactionList == null) {
+            transactionList = new ArrayList<>();
```

Comment:
This is a nitpick, but you could return `Collections.emptyList()` here if you want:
https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()